src/server/scripts/EasternKingdoms/SunkenTemple/sunken_temple.h

```diff
@@ -23,9 +23,49 @@
 #define STScriptName "instance_sunken_temple"
 #define DataHeader "ST"
 
-enum STEvents
+uint32 const EncounterCount = 3;
+
+enum STDataTypes
+{
+    DATA_JAMMAL_AN_THE_PROPHET = 0,
+    DATA_SHADE_OF_ERANIKUS     = 1,
+    DATA_ELITE_TROLLS          = 2,
+
+    EVENT_STATE                = 10
+};
+
+enum STGameObjectIds
+{
+    GO_ATALAI_STATUE1 = 148830,
+    GO_ATALAI_STATUE2 = 148831,
+    GO_ATALAI_STATUE3 = 148832,
+    GO_ATALAI_STATUE4 = 148833,
+    GO_ATALAI_STATUE5 = 148834,
+    GO_ATALAI_STATUE6 = 148835,
+    GO_ATALAI_LIGHT1 = 148883,
+    GO_ATALAI_LIGHT2 = 148937,
```

Comment:
codestyle: keep alignment with the lines above (use the same indentation for enum values within the bracket).